### PR TITLE
Address feedback

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15176/merge/testResults/webgpuplaywright/index.html

File: packages/dev/core/src/Collisions/gpuPicker.ts [hunk 1]
```diff
@@ -26,9 +26,12 @@ export class GPUPicker {
     private _pickableMeshes: Array<AbstractMesh>;
     private _readbuffer: Uint8Array;
     private _meshRenderingCount: number = 0;
-    private _attributeName = "instanceMeshID";
+    private readonly _attributeName = "instanceMeshID";
 
     private _createRenderTarget(scene: Scene, width: number, height: number) {
+        if (this._pickingTexure) {
+            this._pickingTexure!.dispose();
```
Comment (Popov72) [inline packages/dev/core/src/Collisions/gpuPicker.ts:33]:
```suggestion
            this._pickingTexure.dispose();
```

**Review by Popov72 — APPROVED** [review]

**Comment by bjsplat** [comment]
Visualization tests for WebGL 1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15176/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Review by sebavan — APPROVED** [review]

**Review by Popov72 — APPROVED** [review]

**Pull request merged**